clinica/pipelines/t1_volume_dartel2mni/t1_volume_dartel2mni_cli.py

```diff
@@ -27,7 +27,7 @@ def cli(
     group_label: str,
     smooth: List[int] = (8,),
     tissues: List[int] = (1, 2, 3),
-    modulate: bool = False,
+    modulate: bool = True,
```

Comment:
Can't we indicate somehow that there are both modulate and no modulate flags in here?

Reply:
`modulate` here is the variable receiving the boolean value. The flag definitions are in the corresponding `Click.option` which eventually provides its value. Not much to be done further.

Reply:
I added an explicit `"modulate"` target variable name for what it's worth.